pages/company/components/Hero.tsx

```diff
@@ -50,7 +51,12 @@ export const Hero: FC<IHero> = ({}) => {
 						(img, index) => (
 							<SwiperSlide key={index}>
 								<Box sx={{ userSelect: "none" }}>
-									<Image src={img} alt={`Hero Slider Image ${index + 1}`} />
+									<Image
+										src={img}
+										alt={`Hero Slider Image ${index + 1}`}
+										height={330}
+										width={330}
```

Comment:
### **BAD STYLING**

This image is not responsive at all. For different screen sizes, images should be free to have different sizes. 

**By applying those absolute sizes, you are restricting responsiveness.**

Reply:
[This were the required changes](https://github.com/WalkingPal/walking-pal-web/pull/111/commits/5dd8967064183114ad6a89aa1ab6cdb33a537bd8) that you should have applied!

Learn using devtools